server/lib/express/routes/api/v1/users/[user_id]/createSvg.js

```diff
@@ -412,7 +412,7 @@ async function createSvg(userData, options = {}) {
                 font-weight: 500;
                 color: ${variables.colors.text.secondary};
               ">
-                User is not currently doing anything.
+                ${options.noActivityMessageSecondary || 'User is not currently doing anything.'}
```

Comment:
Add it to the defaultOptions object in `server/lib/express/routes/api/v1/users/[user_id]/index.js` file